feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.ts

```diff
@@ -7,28 +14,39 @@ import {
   ProductImportStatus,
   ProductImportSummary,
 } from '@spartacus/cart/base/root';
+
 import {
   FocusConfig,
   ICON_TYPE,
   LaunchDialogService,
 } from '@spartacus/storefront';
+
 import { BehaviorSubject, Observable } from 'rxjs';
 import { finalize, pluck } from 'rxjs/operators';
-
 @Component({
   selector: 'cx-import-entries-dialog',
   templateUrl: './import-entries-dialog.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ImportEntriesDialogComponent {
+export class ImportEntriesDialogComponent implements OnDestroy{
   iconTypes = ICON_TYPE;
   focusConfig: FocusConfig = {
     trap: true,
     block: true,
-    autofocus: 'button',
-    focusOnEscape: true,
+    autofocus: false,
```

Comment:
I suggest using `autofocus: 'button.btn-action'` instead here. Is there a reason why you changed it to `false`?

Reply:
Thanks. autofocus: 'button.btn-action' is the right thing.  It puts the focus on the "Select File" button which is where I wanted it but I did not know how.  

When I set autofocus to true the focus was placed on the dismiss "+" button which is a poor choice. Autofocus false left the dialog unfocused which is nicer.